src/Shared/PackageManagers/BaseProjectManager.cs

```diff
@@ -411,6 +411,13 @@ public virtual List<Version> GetVersions(JsonDocument? metadata)
             throw new NotImplementedException($"{typeName} does not implement GetVersions.");
         }
 
+        /// <inheritdoc />
+        public virtual Task<DateTime?> GetPackageCreatedAtUtcAsync(PackageURL purl, bool useCache = true)
+        {
+            string typeName = GetType().Name;
+            throw new NotImplementedException($"{typeName} does not implement GetPackageCreatedAtUtcAsync.");
+        }
```

Comment:
Should we really add a new interface method & base class implementation if for the vast majority of package managers it just throws an exception? I wonder whether it would make more sense to just implement this on NPMPackageManager for now and to only elevate it to IBaseProjectManager once more package managers add support for it - otherwise it feels like false advertising in a way.

Reply:
I have removed the method in favor of using GetPackageMetadataAsync method to fetch metadata.